src/core/containers/SectionTabs.tsx

```diff
@@ -22,12 +22,16 @@ const SECTION_ITEMS = [
 export class _SectionTabs extends React.Component<any> {
     renderLink({ section, label }) {
         const { activeSection, setSection } = this.props;
+        const style = {
```

Comment:
@ewal2 good idea, a good practice here would be to move the style const out of the render function and to put it in the beginning of the file.